kinto_http/tests/functional.py

```diff
@@ -292,35 +366,39 @@ def test_records_deletion_if_exists(self):
     def test_bucket_sharing(self):
         alice_credentials = ('alice', 'p4ssw0rd')
         alice_userid = self.get_user_id(alice_credentials)
-
         # Create a bucket and share it with alice.
         self.client.create_bucket('shared-bucket',
                                   permissions={'read': [alice_userid, ]})
-
         alice_client = Client(server_url=self.server_url,
                               auth=alice_credentials)
         alice_client.get_bucket('shared-bucket')
 
+    def test_updating_data_on_a_group(self):
+        client = Client(server_url=self.server_url, auth=self.auth,
+                        bucket='mozilla', group='payments')
+        client.create_bucket()
+        client.create_group(data={'members': ['blah', ]})
+        client.patch_group(data={'secret': 'psssssst!', 'members': ['blah', ]})
```

Comment:
Can you remove `members` there? Or add a XXX about: https://github.com/Kinto/kinto/issues/793